api/lib/domain/services/token-service.js

```diff
@@ -93,6 +93,16 @@ function createCertificationResultsLinkToken({ sessionId, daysBeforeExpiration }
   );
 }
 
+function createPasswordResetToken(userId) {
+  return jsonwebtoken.sign(
+    {
+      user_id: userId,
+    },
```

Comment:
Si je ne me trompes pas, ce code retournera toujours le même token pour le même `userId`. Avoir un token différent ne serait pas judicieux ?

```suggestion
{
  user_id: userId,
  uuid: uuidv4(),
}
```

Reply:
J'ai testé avec le même utilisateur et ça retourne des tokens différents à chaque fois.

`eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..Ku7SB8tNTdHxYM5nUn66s4hoXFEd9PL1uOKbZmAP7uY`

`eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..xm96PqNDlmEuAmG8Rq3dSTDNp3ZXpBRWIsSNkubpxeE`

`eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..QregDldB7NXI-6O4AxWIP2xXWogWL2azWwzK7WoGjqo`

Reply:
Oui, parce que la librairie ajoute automatiquement les dates de création (`iat`) et d'expiration (`exp`) au payload du token, ce qui modifie aussi la signature. Cela dit, comme la précision de la date est la seconde, on peut en théorie avoir deux tokens identiques s'ils sont générés à la même seconde (on peut le vérifier en appelant deux fois à la suite la méthode dans le test unitaire). À mon humble avis, dans ce cas précis le risque est tolérable.